src/renderer/viz/expressions/classification/Classifier.js

```diff
@@ -39,14 +48,15 @@ export default class Classifier extends BaseExpression {
             childInlines[this.childrenNames[index]] = source.inline;
         });
         const funcName = `classifier${this.classifierUID}`;
+        const divisor = this.numCategories - 1 || 1;
         const elif = (_, index) =>
             `${index > 0 ? 'else' : ''} if (x<(${childInlines[`arg${index}`]})){
-                return ${index.toFixed(2)};
+                return ${(index / divisor).toFixed(20)};
             }`;
         const funcBody = this.breakpoints.map(elif).join('');
         const preface = `float ${funcName}(float x){
             ${funcBody}
-            return ${this.breakpoints.length.toFixed(1)};
+            return ${(this.breakpoints.length / divisor).toFixed(20)};
```

Comment:
I think this can be replaced by `return 1.;`

Reply:
You are right!